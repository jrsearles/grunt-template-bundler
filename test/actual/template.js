jsMethod("<div data-bind=\"text: value\"/>");
jsMethod("<div>foo</div>");