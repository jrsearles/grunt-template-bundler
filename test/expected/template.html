<div data-bind="text: value"/>
<div>foo</div>